src/mono/System.Private.CoreLib/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,29 @@
+Compat issues with assembly System.Private.CoreLib:
+TypesMustExist : Type 'Internal.Console' does not exist in the reference but it does exist in the implementation.
+TypesMustExist : Type 'Internal.Runtime.InteropServices.ComponentActivator' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'public void System.AppContext.add_FirstChanceException(System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs>)' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'public void System.AppContext.add_ProcessExit(System.EventHandler)' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'public void System.AppContext.add_UnhandledException(System.UnhandledExceptionEventHandler)' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'public void System.AppContext.remove_FirstChanceException(System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs>)' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'public void System.AppContext.remove_ProcessExit(System.EventHandler)' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'public void System.AppContext.remove_UnhandledException(System.UnhandledExceptionEventHandler)' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'public void System.Environment.FailFast(System.String, System.Exception, System.String)' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'protected void System.Diagnostics.Tracing.EventCounter.Flush()' does not exist in the reference but it does exist in the implementation.
+CannotMakeTypeAbstract : Type 'System.Diagnostics.Tracing.EventListener' is abstract in the reference but is not abstract in the implementation.
+CannotMakeMoreVisible : Visibility of member 'System.Diagnostics.Tracing.EventListener..ctor()' is 'protected' in the reference but 'public' in the implementation.
```

Comment:
Why is that?

Reply:
Not sure, but it is a pre-existing issue, see src/libraries/System.Diagnostics.Tracing/src/MatchingRefApiCompatBaseline.txt

Reply:
@ViktorHofer I just tried but since the three CoreLib implementations expose different/additional public APIs in a few places right now it causes quite a mess with the "invalid/Unused baseline difference" logic in ApiCompat that would need conditionally including differerent MatchingRefApiCompatBaseline.txt which gets pretty hard to understand, I don't think it makes sense. Long term we should fix these inconsistencies and then we can do it.

Reply:
Can you please file follow-up issues to a) get the public apis in sync between corelibs and b) honor the MatchingRefApiCompatBaseline.txt files from the various libs?

Reply:
Filed https://github.com/dotnet/runtime/issues/72594 and https://github.com/dotnet/runtime/issues/72595